insoundz_api/src/api.py

```diff
@@ -68,7 +66,7 @@ def account_token(
     def get_default_endpoint_url():
         return DEFAULT_ENDPOINT_URL
 
-    def enhance_file(self, retention=None, preset=None, version=DEFAULT_ENHANCE_VERSION):
+    def enhance_file(self, retention=None, preset=None, enable_stereo=False, version=DEFAULT_ENHANCE_VERSION):
```

Comment:
I think it's always better to add new arguments to the end. Otherwise it can break old implementaion.  
For example if someone calls the old omplemetion of `enhance_file` with `enhance_file(65, 'flat', '3)`.